### ISSUE TITLE
Document instructions for VFS teams re: how to Coordinate with the VA 508 Office to address accessibility findings (in Full Accessibility & 508 Office Audit)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#29924]
## Issue Description
In the Full Accessibility & 508 Office Audit Collab Cycle touchpoint, we tell VFS teams to "Coordinate with the VA 508 Office to address accessibility findings." however we do not provide instructions to VFS with regards to HOW. 

The purpose of this ticket is to document the instructions that the VFS teams need to follow (ServiceNow) in order to coordinate with the VA 508 Office. The instructions will be documented and published in the Accessibility page of Platform Website.

---
## Tasks
- [ ] _What work is necessary for this story to be completed?_

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by shiragoodman** [comment]
Per @pjhill:Hey, so this is not something that I have handled a ton in the past, but my understanding of the process is that you need to do roughly the following:
1. Navigate to https://yourit.va.gov/va from inside the VA network in order to link the service request to your VA email / identity
2. Select the option for 508 Accessibility Audit Request for the ticket
3. Fill out the form with the relevant data for the request. A URL for the office to review and the details of the change the 508 office needs to evaluate. This should include the details of the reason for the audit request. The collaboration cycle feedback likely indicated that you need to request this audit, so you can include those details.

**Comment by shiragoodman** [comment]
@noahgelman do you have any experience with this?

**Comment by noahgelman** [comment]
@shiragoodman I don't. I've never visited yourit.va.gov/va

**Comment by pjhill** [comment]
![508audit](https://user-images.githubusercontent.com/52456633/133507470-6b308da6-6bf1-4df9-83f7-6c1f3badcabb.gif)

**Comment by shiragoodman** [comment]
[https://app.zenhub.com/files/133843125/eb5c5bb5-f125-4191-bbef-7c7136ecf456/download](https://app.zenhub.com/files/133843125/eb5c5bb5-f125-4191-bbef-7c7136ecf456/download)

**Comment by peggygannon** [comment]
Hi, @shiragoodman @noahgelman 

Here is the page with instructions for how teams can submit their product to the VA 508 office. Can you review when you get a chance and let me know if you have any feedback?  Thanks! 

https://vfs.atlassian.net/wiki/spaces/~248144906/pages/1912668266/How+to+submit+your+product+to+the+VA+508+office

**Comment by shiragoodman** [comment]
@peggygannon the instructions look good to me, however I cannot validate them as I don't have access to the VA network.  @pjhill can you help us out please?

Is the plan for these instructions to live alongside the instructions for a [508 office document review?](https://depo-platform-documentation.scrollhelp.site/developer-docs/Instructions-for-VA-508-Office-document-review.1857913061.html) Maybe we can merge these into 1 document titled, "How to request support from the VA 508 Office" and have 1 section covering "Document Review" and the other covering "Audit Request". What do you think?

**Comment by pjhill** [comment]
> @peggygannon the instructions look good to me, however I cannot validate them as I don't have access to the VA network. @pjhill can you help us out please?
> 
> Is the plan for these instructions to live alongside the instructions for a [508 office document review?](https://depo-platform-documentation.scrollhelp.site/developer-docs/Instructions-for-VA-508-Office-document-review.1857913061.html) Maybe we can merge these into 1 document titled, "How to request support from the VA 508 Office" and have 1 section covering "Document Review" and the other covering "Audit Request". What do you think?

I stepped through the instructions, and they're good. I added some screen shots of the elements the instructions call out. You can keep them or remove them. I figured I'd capture them while stepping through the instructions.

**Comment by shiragoodman** [comment]
@pjhill thank you! Since Peggy is out, I'm going to wrap up this work and close it out. I merged the document review doc and the audit request doc into 1 and published to Platform Website: https://depo-platform-documentation.scrollhelp.site/developer-docs/Instructions-for-VA-508-Office-document-review.1857913061.html
Can you check it out? I'll be linking to this page from the 508 Audit touchpoint page as well as from the 508 Audit gh template (but needed to create the page before I can add links). Let me know!
